store/migration/10002__alter_vcs_type_check.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE vcs DROP CONSTRAINT vcs_type_check;
+ALTER TABLE vcs ADD CONSTRAINT vcs_type_check CHECK (type IN ('GITLAB_SELF_HOST', 'GITHUB_DOT_COM'));
```

Comment:
There are couple other places we need to do this in the schema, just search GITLAB_SELF_HOST